### ISSUE TITLE
Enable SQL inline comments

### ISSUE THREAD

**Issue by garrett-gottlieb** [orientechnologies/orientdb#2675]
It's very useful to have comments on the same line as the sql statement. For example,

```sql
CREATE CLASS ClaimedPromoCode EXTENDS E -- OUser -> PromoCode
CREATE CLASS Referred EXTENDS E -- OUser -> OUser
CREATE CLASS HasSettings EXTENDS E -- OUser -> Settings
```

**Comment by gongdo** [comment]
It's a year ago, but I still struggle with comments when I'm using Studio.
Any update about this, please?

About comment style, well-known comment style would be nicer like:
`// comment`, `/* multi-line comments */, `-- sql style comment`
If I should choose only one of these however, I would go for `--`.

**Comment by luigidellaquila** [comment]
Hi guys,

it's definitely time to fix this problem. I'm working on it right now, I
guess I'll be able to release a fix on 2.1.x and develop branches in a few
hours (new parser only, so no backporting to 2.0.x, sorry)

Thanks

Luigi


2015-10-13 7:58 GMT+02:00 gongdo <notifications@github.com>:

**Comment by luigidellaquila** [comment]
Fixed in 2.1.x and develop branches, now multi-line java style comments are supported, 
eg. /* comment */

The behavior is consistent with OrientDB studio editor ctrl+/
